Justfile

```diff
@@ -17,6 +17,9 @@ check-mypy:
     MYPYPATH=probe_src mypy --strict --package probe_py
     mypy --strict probe_src/libprobe
 
+check-clang-tidy:
+    clang-tidy probe_src/**/*.c -- -Iinclude
+
```

Comment:
We should also check `.h`

Reply:
yeah,, will do this. can you please tell why are the build checks currently failing?